optimum/exporters/ipex/model_patcher.py

```diff
@@ -73,13 +92,10 @@ def _patch_llama_model(model):
         raise ImportError(
             f"Only transformers versions {_TRANSFORMERS_MIN_VERSION} ~ {_TRANSFORMERS_MAX_VERSION} are verified."
         )
-    convert_functions(model, LlamaModel, "forward", _llama_model_forward)
-    convert_functions(model, LlamaRMSNorm, "forward", _llama_layer_norm_forward)
-    convert_class(model, LlamaDecoderLayer, _IPEXLlamaDecoderLayer, model.config)
-    return model
-
-
-def _patch_model(model):
     if isinstance(model, LlamaForCausalLM):
         model = _patch_llama_model(model)
+    elif isinstance(model, BertForQuestionAnswering):
```

Comment:
why not check the `model_type` from the config instead https://huggingface.co/google-bert/bert-base-uncased/blob/main/config.json#L14 ?  (so that it's for all bert architecture not only for QA)